spack-environments/csd3/spack.yaml

```diff
@@ -4,7 +4,7 @@
 # configuration settings.
 spack:
   # add package specs to the `specs` list
-  specs: []
+  specs: [sombrero@2021-08-16, intel-mpi-benchmarks@2019.6]
```

Comment:
These must have been updated automatically by spack. Should I remove them?

Reply:
Yeah, I think that should be better.  I'm still not 100% sure about what this though: I'd like to keep it empty by default in case we end up having incompatible packages from different benchmarks, but hopefully that won't happen.